receiver/sqlqueryreceiver/README.md

```diff
@@ -73,6 +87,13 @@ receivers:
   sqlquery:
     driver: postgres
     datasource: "host=localhost port=5432 user=postgres password=s3cr3t sslmode=disable"
+    datasource_config:
+      host: localhost
+      port: 5432
+      username: postgres
+      password: s3cr3t
+      additional_params:
+        sslmode: disable
```

Comment:
Other SQL receivers (MySQL, PostgreSQL, SQL Server) offer all of these options as top-level options (without being under the `datasource_config` option.) The SQL Server receiver offers them alongside each other.

What do you think of moving these all up so they're not under `datasource_config`? Any qualms with that approach? It would be good to be as consistent with other receivers.